docker/testrunner/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
+RUN apt-get install -y sudo openjdk-7-jdk xorg xvfb xserver-xorg-video-dummy
+RUN useradd -m -G users testrunner
+ADD entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
```

Comment:
Still this `chmod` would be executed when building the image and not inside the container with the mounted folder.